pycti/utils/opencti_stix2.py

```diff
@@ -1623,31 +1624,38 @@ def prepare_export(
         )
         for stix_nested_ref_relationship in stix_nested_ref_relationships:
             if "standard_id" in stix_nested_ref_relationship["to"]:
-                if MultipleRefRelationship.has_value(
+                # dirty fix because the sample and operating-system ref are not multiple for a Malware Analysis
```

Comment:
Still needed ?

Reply:
Yes! Agreed with @richard-julien to do it this way for the moment